_translations/en.yml

```diff
@@ -586,9 +586,9 @@ en:
     title: "Bitcoin - Open source P2P money"
     metadescription: "Bitcoin is an innovative payment network and a new kind of money. Find all you need to know and get started with Bitcoin on bitcoin.org."
     listintro: "Bitcoin is an innovative payment network and a new kind of money."
-    list1: "Fast peer-to-peer<br>transactions"
+    list1: "Uncensorable peer-to-peer<br>transactions"
     list2: "Worldwide<br>payments"
-    list3: "Low<br>processing fees"
+    list3: "No chargebacks"
```

Comment:
I like this substitute.

Reply:
I like no chargebacks as it is easily understandable by non-technical people but adequately describes one of the important properties of bitcoin transactions.

Reply:
In some situation 'no chargebacks' is a downside, not an upside because of the lack of consumer protection.
If you're going for this angle a better option might be 'no chargebacks (but consumer protection is still possible)' with a link to a page about multisignature and escrow. I used this phrasing when working on [the bitcoin wiki page](https://en.bitcoin.it/wiki/Bitcoin_as_a_medium_of_exchange#Bitcoin_as_a_low-trust_medium_of_exchange)